integrations/tensorflow/bindings/python/iree/tf/support/module_utils.py

```diff
@@ -89,12 +89,10 @@ def _get_tf_import_output_kwargs(artifacts_dir: str,
   return kwargs
 
 
-def _incrementally_compile_tf_module(
-    module: Type[tf.Module],
-    backend_info: "BackendInfo",
-    exported_names: Sequence[str] = (),
-    artifacts_dir: Optional[str] = None,
-) -> Tuple[bytes, Optional[str]]:
+def _incrementally_compile_tf_module(module: Type[tf.Module],
+                                     backend_info: "BackendInfo",
+                                     exported_names: Sequence[str] = (),
+                                     artifacts_dir: Optional[str] = None):
```

Comment:
```suggestion
def _incrementally_compile_tf_module(
    module: Type[tf.Module],
    backend_info: "BackendInfo",
    exported_names: Sequence[str] = (),
    artifacts_dir: Optional[str] = None,
) -> Tuple[bytes, Optional[str]]:
```
Nit: keep the output type signature.